src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/OperatorSymbolExtensions.cs

```diff
@@ -0,0 +1,217 @@
+                WellKnownMemberNames.SubtractionOperatorName => "-",
+
+                // Unary
+                WellKnownMemberNames.DecrementOperatorName => "--",
+                WellKnownMemberNames.FalseOperatorName => "false",
+                WellKnownMemberNames.IncrementOperatorName => "++",
+                WellKnownMemberNames.LogicalNotOperatorName => "!",
+                WellKnownMemberNames.OnesComplementOperatorName => "~",
+                WellKnownMemberNames.TrueOperatorName => "true",
+                WellKnownMemberNames.UnaryNegationOperatorName => "-",
+                WellKnownMemberNames.UnaryPlusOperatorName => "+",
+
+                var name => throw ExceptionUtilities.UnexpectedValue(name),
+            };
+        }
+
+        internal static int GetOperatorSortIndex(this IMethodSymbol m)
+        {
+            return m.Name switch
+            {
+                // comparison and negation
+                WellKnownMemberNames.EqualityOperatorName => 0, // ==
+                WellKnownMemberNames.InequalityOperatorName => 1, // !=
+                WellKnownMemberNames.GreaterThanOperatorName => 2, // >
+                WellKnownMemberNames.GreaterThanOrEqualOperatorName => 3, // >=
+                WellKnownMemberNames.LessThanOperatorName => 4, // <
+                WellKnownMemberNames.LessThanOrEqualOperatorName => 5, // <=
+                WellKnownMemberNames.LogicalNotOperatorName => 6, // !
+                // mathematical
+                WellKnownMemberNames.AdditionOperatorName => 7, // +
+                WellKnownMemberNames.SubtractionOperatorName => 8, // -
+                WellKnownMemberNames.MultiplyOperatorName => 9, // *
+                WellKnownMemberNames.DivisionOperatorName => 10, // /
+                WellKnownMemberNames.ModulusOperatorName => 11, // %
+                WellKnownMemberNames.IncrementOperatorName => 12, // ++
+                WellKnownMemberNames.DecrementOperatorName => 13, // --
+                WellKnownMemberNames.UnaryPlusOperatorName => 14, // +
+                WellKnownMemberNames.UnaryNegationOperatorName => 15, // -
+                // bit operations
+                WellKnownMemberNames.BitwiseAndOperatorName => 16, // &
+                WellKnownMemberNames.BitwiseOrOperatorName => 17, // |
+                WellKnownMemberNames.ExclusiveOrOperatorName => 18, // ^
+                WellKnownMemberNames.LeftShiftOperatorName => 19, // <<
+                WellKnownMemberNames.RightShiftOperatorName => 20, // >>
+                WellKnownMemberNames.OnesComplementOperatorName => 21, // ~
+                // true false
+                WellKnownMemberNames.FalseOperatorName => 22, // false
+                WellKnownMemberNames.TrueOperatorName => 23, // true
+
+                var name => throw ExceptionUtilities.UnexpectedValue(name),
+            };
+        }
+
+        public static string GetOperatorName(this IMethodSymbol m)
```

Comment:
This method is not used yet, but could be used in `CompletionItem.InnerDescription` and displayed right-aligned in the completion list (like the namespaces of unimported extensions methods).

Reply:
if it's not used, plz remove.  If it is useful, then plz use it :)

Reply:
I gave it a try in 517f59f. I also updated the screencast to show how it looks like. I don't think it is working well. It looks too noisy for me.

Reply:
The method is in use now. It should be renamed because`GetOperatorName` is misleading.